stubs/fpdf2/fpdf/util.pyi

```diff
@@ -1,13 +1,24 @@
 from collections.abc import Iterable
-from typing import Any, AnyStr
+from typing import Any, AnyStr, NamedTuple
 from typing_extensions import Final, Literal, TypeAlias
 
 _Unit: TypeAlias = Literal["pt", "mm", "cm", "in"]
 
 PIL_MEM_BLOCK_SIZE_IN_MIB: Final = 16
 
+class Padding(NamedTuple):
+    # TODO: replaced Number with float
```

Comment:
```suggestion
    # TODO: replace Number with float
```

Reply:
What I meant here is that `stubgen` created this with `Number` and I replaced it with `float` (`int | float` caused a violation that suggests to only use `float`). 

I can revert to `Number` or remove the TODO.

Reply:
Removing the default `= ...` caused the following error:

```
error: fpdf.util.Padding.__new__ is inconsistent, runtime argument "top" has a default value but stub argument does not
Stub: in file /home/runner/work/typeshed/typeshed/stubs/fpdf2/fpdf/util.pyi:9
def [_NT <: tuple[builtins.float, builtins.float, builtins.float, builtins.float]] (_cls: Type[_NT`-1], top: builtins.float, right: builtins.float, bottom: builtins.float, left: builtins.float) -> _NT`-1
Runtime:
def (_cls, top: numbers.Number = 0, right: numbers.Number = 0, bottom: numbers.Number = 0, left: numbers.Number = 0)
```

So I switched back to `Number`.

Reply:
Hm, I can't get it to work. What's the correct way to reflect the implementation?

I've tried:

```py
top: Number
top: Number = ...
top: Number = 0
```

Reply:
@mschoettle The annotations in the upstream source code using `Number` are a bit weird. Let me experiment for a bit.